src/main/kotlin/uk/gov/justice/digital/hmpps/pollpush/config/JmsConfig.kt

```diff
@@ -25,14 +25,14 @@ import javax.jms.Session
 
 @Configuration
 @EnableJms
-open class JmsConfig {
+class JmsConfig {
   companion object {
     val log: Logger = LoggerFactory.getLogger(this::class.java)
   }
 
   @Bean
   @Suppress("SpringJavaInjectionPointsAutowiringInspection")
-  open fun jmsListenerContainerFactory(awsSqsClient: AmazonSQS): DefaultJmsListenerContainerFactory {
+  fun jmsListenerContainerFactory(@Qualifier("awsSqsClient") awsSqsClient: AmazonSQS): DefaultJmsListenerContainerFactory {
```

Comment:
We have two clients since one dead letter one.  Not sure why adding a integration test exposed it though